plone/app/contenttypes/migration/migration.py

```diff
@@ -343,10 +348,6 @@ class FolderMigrator(ATCTFolderMigrator):
     dst_portal_type = 'Folder'
     dst_meta_type = None  # not used
 
-    def beforeChange_migrate_layout(self):
-        if self.old.getLayout() == 'atct_album_view':
-            self.old.setLayout('album_view')
```

Comment:
There is a mapping defined for atct_album_view in https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/upgrades.py#L15
So this shouldn't be necessary and handled by the ATCTFolderMigrator code above.